src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -3493,6 +3502,49 @@ - (void)removeClientDataForKey:(NSString *)key endpointID:(NSNumber *)endpointID
     [self.temporaryMetaDataCache removeObjectForKey:[NSString stringWithFormat:@"%@:%@", key, endpointID]];
 }
 
+#pragma mark Log Help
+
+- (NSNumber *)_informationalNumberAtAttributePath:(MTRAttributePath *)attributePath
+{
+    auto * cachedData = [self _cachedAttributeValueForPath:attributePath];
+
+    auto * attrReport = [[MTRAttributeReport alloc] initWithResponseValue:@{
+        MTRAttributePathKey : attributePath,
+        MTRDataKey : cachedData,
+    }
+                                                                    error:nil];
+    // REVIEWERS:  is it worth logging the `error` above?
```

Comment:
Error here would be "we expected a number but what we got is not a number", right?

I think this method should return `nullable NSNumber *` to make that clear.

Past that, logging it might not be worth it.